Framework/Core/include/Framework/ExternalFairMQDeviceProxy.h

```diff
@@ -56,12 +56,29 @@ InjectorFunction incrementalConverter(OutputSpec const& spec, uint64_t startTime
 /// multipart ensemble.
 InjectorFunction o2DataModelAdaptor(OutputSpec const& spec, uint64_t startTime, uint64_t step);
 
+/// @struct DPLModelAdapterConfig
+/// Configuration object for dplModelAdaptor
+struct DPLModelAdapterConfig {
+  /// throw runtime error if an input message is not matched by filter rules
+  bool throwOnUnmatchedInputs = true;
+  /// do all kinds of consistency checks
+  bool paranoid = false;
+  /// blindly forward on one channel
+  bool blindForward = false;
+};
```

Comment:
Maybe this should be an enum?

Reply:
it's not just definitions of options but we can have multiple active options at the same time. That would mean to bind the enum definitions to bits and then probing for the bits. But true, I wanted to work on a constructor with a flexible argument list to set the members, got a bit out of scope.

Reply:
I'm going to implement to paranoid checks and forward optimization in a coming commit, and then I will also add the constructor.